specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/2016-06-01/examples/ListOperations.json

```diff
@@ -0,0 +1,1339 @@
+                            "Resource": "Vaults",
+                            "Operation": "GetContainers",
+                            "Description": "The Get Containers operation can be used get thecontainers registered for a resource."
+                        },
+                        "Origin": "user,system"
+                    },
+                    {
+                        "Name": "Microsoft.RecoveryServices/Vaults/registeredIdentities/delete",
+                        "Display": {
+                            "Provider": "Microsoft.RecoveryServices",
+                            "Resource": "Vaults",
+                            "Operation": "UnregisterService Container",
+                            "Description": "The UnRegister Container operation can beused to unregister a container."
+                        },
+                        "Origin": "user,system"
+                    },
+                    {
+                        "Name": "Microsoft.RecoveryServices/Vaults/certificates/write",
+                        "Display": {
+                            "Provider": "Microsoft.RecoveryServices",
+                            "Resource": "Vaults",
+                            "Operation": "UpdateResource Certificate",
+                            "Description": "The Update Resource Certificate operationupdates the resource/vault credential certificate."
+                        },
+                        "Origin": "user"
+                    },
+                    {
+                        "Name": "Microsoft.RecoveryServices/Vaults/monitoringAlerts/read",
+                        "Display": {
+                            "Provider": "Microsoft.RecoveryServices",
+                            "Resource": "Vaults",
+                            "Operation": "Getalerts",
+                            "Description": "Gets the alerts for the Recovery services vault."
+                        },
+                        "Origin": "user"
+                    },
+                    {
+                        "Name": "Microsoft.RecoveryServices/Vaults/monitoringAlerts/{uniqueAlertId}/read",
+                        "Display": {
+                            "Provider": "Microsoft.RecoveryServices",
+                            "Resource": "Vaults",
+                            "Operation": "Getalert details",
+                            "Description": "Gets the details of the alert."
+                        },
+                        "Origin": "user"
+                    },
+                    {
+                        "Name": "Microsoft.RecoveryServices/Vaults/monitoringAlerts/{uniqueAlertId}/patch",
+                        "Display": {
+                            "Provider": "Microsoft.RecoveryServices",
+                            "Resource": "Vaults",
+                            "Operation": "Resolvealert",
+                            "Description": "Resolves the alert."
+                        },
```

Comment:
There is a remaining issue with the example model validation https://travis-ci.org/Azure/azure-rest-api-specs/jobs/259348817#L657 
The value for blobDuration doesn't match the date-time format per https://xml2rfc.tools.ietf.org/public/rfc/html/rfc3339.html#anchor14
Please update these.